gazebo_ros/test/test_plugins.cpp

```diff
@@ -34,94 +34,89 @@ struct TestParams
 class TestPlugins : public ::testing::TestWithParam<TestParams>
 {
 public:
-  // Documentation inherited
+  /// Documentation inherited
```

Comment:
The reason why it was just a normal comment was to indicate that doxygen should pull the documentation from the base function if available. By changing it to `///` we're overriding the original documentation with "Documentation inherited", which is probably not what we want to do.